src/preferences_window.py

```diff
@@ -342,7 +342,7 @@ def create_colors_page(self):
         self.profile_new_button_add.connect('clicked', \
             self.create_color_profile)
         self.profile_new_box.append(self.profile_new_button_add)
-        self.profile_new_label = Gtk.Label.new('')
+        self.profile_new_label = Gtk.Label.new(_('The new profile will be a copy of the active profile.'))
```

Comment:
The same change is required on line 626

Reply:
Ah, yes, I thought I was missing something.

I can fix it in a couple of hours, or you can if you want.